### DISCUSSION TITLE
Welcome to RiverscapesXML Discussions!

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#88] [General] [answered]
This is really a place to share ideas related to symbolizing riverscapes projects as the RiverscapesXML is where those project and model owners define the business logic and symbology to help people see it.

**Comment by joewheaton** [comment, marked as answer]
Hey @Riverscapes/riverscape-project-curators and @Riverscapes/etal-fhc folks! Thanks for a great workshop today on Riverscape Project Curation and Cartography. You guys were troopers. I have added each of you to this Riverscape Project Curators team so we can use the `@Riverscapes/riverscape-project-curators` mention to let you know about things of interest to all of you. Please use these discussion boards. @MattReimer has been busy providing you helpful discussion announcements [here](https://github.com/Riverscapes/RiverscapesXML/discussions) (we failed to mention those today) in addition to the documentation we have at [http://rave.riverscapes.xyz](http://rave.riverscapes.xyz). 

A few reminders from today for your reference:
- @wally-mac, @philipbaileynar and I will be assigning you issues and tasks (` - [ ]`) within those issues that reflect divvying up these curation tasks amongst the group. The @Riverscapes/etal-fhc will be responsible for  Riverscapes Context, RCAT (old and new), pyBRAT and sqlBRAT, and RIM.  @lauren-herbine will be in charge of Confinement and some new River Styles project types. @shelbysawyer will be in charge of VBET. @nick4rivers and @CHafen will be in charge of Riverscapes Studio and its children (e.g. LTPBR stuff). **Our goal is to finish v 1.0 of completely curated project types within the next 2 months.** We can get there if we help each other out.
- If you want to have a play on the not to be shared broadly webRAVE [here's the link](https://northarrowresearchlabs.github.io/riverscapes-staging/#/) (request access from @MattReimer). This is where you will test webRAVE symbology BTW
- The @Riverscapes/etal-fhc group (mainly @tyler1218hatch) will be working to get legacy pyBRAT and RCAT projects up into the warehouse.
- See [here](https://github.com/Riverscapes/RiverscapesXML/discussions/categories/rave-symbology-businesslogic) for discussions @MattReimer has posted with regards to Symbology/Business Logic. The difference between these discussion boards and the issue boards are that discussion boards for posting announcements, ideas, questions and brainstorming, where as the issue board is where  bugs get logged, enhancements get requested, etc. Issues get assigned, whereas discussions are just open.

**Reply by nick4rivers** [reply]
I like the direction here @joewheaton and thanks for pointing out these resources.

**Reply by CHafen** [reply]
Where can we watch the recording of the workshop?

**Reply by joewheaton** [reply]
> Where can we watch the recording of the workshop?

I'm checking with Wes James to see if he can send me recording.... it was still downloading Friday night. Finger's crossed.

**Reply by joewheaton** [reply]
I don't have good enough internet here to download right now, but I will in next few days and upload to YouTube. For now, you can download [from BOX if you really want to the MP4](https://usu.box.com/s/lvl2vn4nj2cwrk0ix82kccspqkwyc8pa)...
